code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -804,6 +804,43 @@
 	log_game("[key_name(obsessed)] was made Obsessed by the midround ruleset.")
 	return TRUE
 
+/// Midround Renegade Ruleset (From Living)
+/datum/dynamic_ruleset/midround/from_living/renegade
+	name = "Renegade"
+	midround_ruleset_style = MIDROUND_RULESET_STYLE_LIGHT
+	antag_datum = /datum/antagonist/renegade
+	antag_flag = ROLE_RENEGADE
+	restricted_roles = list(
+		JOB_AI,
+		JOB_CYBORG,
+		ROLE_POSITRONIC_BRAIN,
+	)
+	required_enemies = list(2,2,1,1,1,1,1,0,0,0)
```

Comment:
Do you know what this is doing? I don't know why you're setting it

Reply:
> Do you know what this is doing? I don't know why you're setting it

It was set like this in the Obssesed midround injection, so I thought it is kinda important..?